index.html

```diff
@@ -2,82 +2,304 @@
-                        project you need coded, don't hesitate to contact me.
-                    </p>
-                </div>
-            </div>
-
-            <!-- social media icons-->
-            <ul class="icons-container">
-                <li>
-                    <a href="#main"><img src="./images/git.svg" class="social-media-icon" /></a>
-                </li>
-                <li>
-                    <a href="#main"><img src="./images/linkdin.svg" class="social-media-icon" /></a>
-                </li>
-                <li>
-                    <a href="#main"><img src="./images/angellist.svg" class="social-media-icon" /></a>
-                </li>
-                <li>
-                    <a href="#main"><img src="./images/twitter.svg" class="social-media-icon " /></a>
-                </li>
-                <li>
-                    <a href="#main"><img src="./images/medium.svg" class="social-media-icon" /></a>
-                </li>
-            </ul>
-
-            <!--illustrations-->
-            <div class="illustration-header" aria-hidden="true"></div>
-
-        </section>
-    </main>
+
+  <!--nav Start-->
+  <header class="header" id="header">
+
+    <a href="#">Kirti Arora</a>
+
+    <!-- toggle button -->
+    <nav>
+      <button type="button" class="menu-btn">
+        <span class="open" id="menu"></span>
+      </button>
+    </nav>
+
+  </header>
+  <!--nav End-->
+  <!-- ----Home---------- -->
+  <section class="home" id="home">
+    <div class="home-page">
+      <div>
+        <h1 class="home_header">Hey There.</h1>
+        <h1 class="home_header">I'm Kirti</h1>
+      </div>
+
+      <div class="home-description">
+        <h3 class="intro-heading">I'm a Software Developer</h3>
+
+        <p class="intro-para">
+          I can help you build a product, feature or website Look through some
+          of my work and experiece! If you like what you see and have a
+          project you need coded, don't hesitate to contact me.
+        </p>
+      </div>
+    </div>
+
+    <!-- social media icons-->
+    <ul class="icons-container">
+      <li>
+        <a href="#main"><img src="./images/git.svg" class="social-media-icon" /></a>
+      </li>
+      <li>
+        <a href="#main"><img src="./images/linkdin.svg" class="social-media-icon" /></a>
+      </li>
+      <li>
+        <a href="#main"><img src="./images/angellist.svg" class="social-media-icon" /></a>
+      </li>
+      <li>
+        <a href="#main"><img src="./images/twitter.svg" class="social-media-icon " /></a>
+      </li>
+      <li>
+        <a href="#main"><img src="./images/medium.svg" class="social-media-icon" /></a>
+      </li>
+    </ul>
+
+    <!--illustrations-->
+    <div class="illustration-header" aria-hidden="true"></div>
+
+  </section>
+  <!-- work section -->
+  <section class="work-section-container">
+    <div class="work-title-card">
+      <h1 class="work-title-heading">My Recent Works</h1>
```

Comment:
- Kindly make sure you have only one h1 tag in you all HTML, this is for conventional reasons, h1 one tag is only used for the general title, the title (e.g., "My Recent Works") uses a header tag hx (h2,h3,h5....) :+1: